### ISSUE TITLE
Model weights

### ISSUE THREAD

**Issue by charlielito** [SocialComplexityLab/life2vec#2]
Are you planning to release the model weights?

**Comment by SocialComplexityLab** [comment]
We want to share as much data as possible – to be as open as possible, but we also have to ensure the privacy of people in the study, so the answer is “perhaps”: we need to do some more work to understand what we we can share and what we cannot.

From: Carlos Andrés Álvarez Restrepo ***@***.***>
Date: Tuesday, December 26, 2023 at 6:58 PM
To: SocialComplexityLab/life2vec ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [SocialComplexityLab/life2vec] Model weights (Issue #2)

Are you planning to release the model weights?

**Comment by matiasfreitas** [comment]
I would like to have also the model weights. The dataset would be amazing, but I think it isn't possible due to privacy concerns.

**Comment by carlomarxdk** [comment]
There are no plans to publicly release the model weights, since if you have them and look at the gradients of the model - you can reconstruct chunks of the training dataset. 
We might release weights of the follow up models if we manage to include some kind of privacy preserving mechanism during the pretraining stage - but so far, life2vec model does not have any. 
Meanwhile, you can request an access to the model weights and data via https://www.dst.dk/en/TilSalg/Forskningsservice/Dataadgang.

You can find the simple implementation of the model with the dummy data at https://github.com/carlomarxdk/life2vec-light.
